### PR TITLE
Feature/tdq 13103 percentage in big decimal

### PR DESCRIPTION
**Pull request by jteuladedenantes**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [x] Tests for the changes have been added (for bug fixes / features) ?
- [x] Docs have been added / updated (for bug fixes / features) ?
- [x] The code coverage on new code >75%
- [x] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TDQ-13103
We don't deal with percentages in BigDecimal parser.
For example,  percentages number as **45%** return a NumberFormatException.

**What is the new behavior?**

We deal with percentages in BigDecimal parser. 
**45%** will be recognized as the number **0.45**.

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/154/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/154/)

File: daikon/src/test/java/org/talend/daikon/number/BigDecimalFormatterTest.java [hunk 1]
```diff
@@ -9,50 +9,66 @@
 public class BigDecimalFormatterTest {
 
     @Test
-    public void testToBigDecimal_US() throws Exception {
-        assertEquals("12.5", BigDecimalFormatter.format(new BigDecimal(12.50), BigDecimalParser.US_DECIMAL_PATTERN));
-        assertEquals("12", BigDecimalFormatter.format(new BigDecimal(12), BigDecimalParser.US_DECIMAL_PATTERN));
-        assertEquals("12.58", BigDecimalFormatter.format(new BigDecimal(12.57708), BigDecimalParser.US_DECIMAL_PATTERN));
+    public void testToBigDecimalUS() throws Exception {
+        assertEquals("12.5", BigDecimalFormatter.format(new BigDecimal("12.50"), BigDecimalParser.US_DECIMAL_PATTERN));
```
Comment (sgandon) [inline daikon/src/test/java/org/talend/daikon/number/BigDecimalFormatterTest.java]: Why did you change this tests ? It it suppose to formlat a BigDecimal into a String, so I guess the input value should be BigDecimal and not string.

Comment (jteuladedenantes) [inline daikon/src/test/java/org/talend/daikon/number/BigDecimalFormatterTest.java]: It comes from the following codacy rule:
Avoid creating BigDecimal with a decimal (float/double) literal. Use a String literal
https://www.codacy.com/app/Talend/daikon/pullRequest?prid=803669

But i can undo it if we estimate this codacy rule is not important ;-)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/163/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/163/)
